harmony-fe/src/pages/profile/profile.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {useState } from 'react'
+import { Avatar, Button, Card, Container, Grid, Paper, Tab, Tabs, Typography } from '@mui/material'
+import HarmonyAppBar from '../../components/appbar'
+
+const HomePage = () => {
+  const [currentTabIndex, setCurrentTabIndex] = useState(0);
+ 
+  const handleTabChange = (e: any, tabIndex: React.SetStateAction<number>) => {
+    setCurrentTabIndex(tabIndex);
+  };
+
+  return (
+    <>
+      <HarmonyAppBar />
+      <Container maxWidth="xl">
+      <Paper
+      sx={{
+        p: 2,
+        margin: 'auto',
+        maxWidth: 'auto',
+        flexGrow: 1,
+       }}>
+
+        <Grid container spacing={2} direction="row" justifyContent="flex-end" alignItems="center">
+          <Grid item>
+            <Avatar src="/harmony1.png" sx={{height: '200px', width: '200px' }}></Avatar>
+          </Grid>
+          <Grid item xs={12} sm container>
+            <Grid item xs container direction="column" spacing={2}>
+              <Grid item xs>
+                <Typography gutterBottom variant="h4" component="div">
+                  Screen Name
+                </Typography>
+                <Typography variant="h5" gutterBottom>
+                  User Name
+                </Typography>
+              </Grid>
+            </Grid>
+            <Grid item>
+              <Button className = 'followButton'>Follow</Button>
+              <br/>
+              100 Trillion Followers
+            </Grid>
+          </Grid>
+        </Grid>
+      </Paper>
+
+      <Paper
+      sx={{
+        p: 2,
+        margin: 'auto',
+        maxWidth: 'auto',
+        flexGrow: 1,
+       }}>
+
+        <Grid container spacing={2} direction="row" justifyContent="flex-end" alignItems="center">
+          <Grid item xs={8} container direction="column">
+            <Grid item xs>
+              <h1>Content</h1>
```

Comment:
General comment, not asking for changes or anything but this made me think of it, we should probably build a styled component of of the MUI Typography component where we select the font for our website, and then use that component for all text, that way we can easily make changes to all text on the site by modifying that component, and since its based off of Typography we will still have the flexibility to choose h1, etc.

Reply:
Made an issue for this, #36, we can pick it up when one of us is looking for something to do